src/utils/common.js

```diff
@@ -17,4 +17,28 @@ const classNames = (names) => {
 
 }
 
-export { classNames };
\ No newline at end of file
+const webShare = (data) => {
+    if (navigator.share) { 
+        navigator.share({
+           title: data.title,
+           text: data.text,
+           url: data.url
+        }).then(() => {
+            if (typeof data.successCallback === 'function') {
+                data.successCallback();
+            }
+        }).catch(console.error);
```

Comment:
seems like it's more robust not just log error, but also do fallback method (i.e. copy)?